enlace/programs/profile.php

```diff
@@ -46,6 +48,9 @@
         $('#search_parti_table').hide();
         $('.hide_dates').hide();
         $('.add_new_person_to_session').hide();
+        $('tr.oldsession').hide();
```

Comment:
Throughout this file I'm not sure what these "oldsession" things refer to -- an explanatory comment where they're first declared would help.

Reply:
:+1: